### PR TITLE
Added remove_invalid_parentheses.py in Backtracking

### PR DESCRIPTION
**Pull request by pratham3012**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: backtracking/remove_invalid_parentheses.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    Below is the Python3 implementation of the above problem.
+"""
+
+"""
+    'isParenthesis' Method checks if character is parenthesis(open or closed)
+"""
+
+
+def isParenthesis(c):
```
[the remark this answers is not in the document]
Comment (pratham3012) [reply backtracking/remove_invalid_parentheses.py]: The name of the function has been changed to `is_parenthesis` from `isParenthesis`

File: backtracking/remove_invalid_parentheses.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    return (c == "(") or (c == ")")
+
+
+"""
+    'isValidString' method returns true if contains valid parenthesis
+"""
+
+
+def isValidString(str):
```
[the remark this answers is not in the document]
Comment (pratham3012) [reply backtracking/remove_invalid_parentheses.py]: The name of the function has been changed to `is_valid_string ` from `isValidString `

File: backtracking/remove_invalid_parentheses.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            return False
+    return cnt == 0
+
+
+"""
+    'removeInvalidParenthesis method to remove invalid parenthesis
+"""
+
+
+def removeInvalidParenthesis(str):
```
[the remark this answers is not in the document]
Comment (pratham3012) [reply backtracking/remove_invalid_parentheses.py]: The name of the function has been changed to `remove_invalid_parenthesis ` from `removeInvalidParenthesis `

File: backtracking/remove_invalid_parentheses.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    Below is the Python3 implementation of the above problem.
+"""
+
+"""
+    'is_parenthesis' Method checks if character is parenthesis(open or closed)
+"""
+
+
+def is_parenthesis(c):
```
[the remark this answers is not in the document]
Comment (pratham3012) [reply backtracking/remove_invalid_parentheses.py:19]: The function returns the following value
`return (c == "(") or (c == ")`

File: backtracking/remove_invalid_parentheses.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+            return False
+    return cnt == 0
+
+
+"""
+    'remove_invalid_parenthesis method to remove invalid parenthesis
+"""
+
+
+def remove_invalid_parenthesis(str):
```
[the remark this answers is not in the document]
Comment (pratham3012) [reply backtracking/remove_invalid_parentheses.py:45]: The function return null value.

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that have failing tests.

**Pull request closed without merging**